Solutions/Radiflow/Analytic Rules/RadiflowPlatformAlert.yaml

```diff
@@ -0,0 +1,84 @@
+﻿id: ff0c781a-b30f-4acf-9cf1-75d7383d66d1
+name: Radiflow - Platform Alert
+description: |
+  'Generates an incident when an alert raised by Radiflow's iSID is not contemplated by any of the other analytic rules'
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: RadiflowIsid
+    dataTypes:
+      - RadiflowEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:

```

Comment:
relevant techniques missing here

Reply:
Hi,
RadiflowPlatformAlert: there are no techniques associated with this rule on purpose, but I believe the issue is the relevantTechniques field 'should' be present, even if no techniques have been defined, but I am not 100% sure. I have seen examples of other customers whose relevantTechniques field is missing as well as no techniques have been defined for the analytic rule.
Maybe further clarification is available?

Thank you,